corona-warn-app/cwa-app-android
# Adjust PPDD error dialog messages (EXPOSUREAPP-5201)

Refactor SafetyNetException.kt and SurveyException.kt to use `HasHumanReadableError` interface and fix error codes not being added to the localized strings.

### Testing
* Check the code
* Check the dialogs in the test menu
